pages/integrations/phabricator.md.erb

```diff
@@ -0,0 +1,85 @@
+Build Plans". Press "Create Build Plan" located in the upper right.
+
+<%= image "phabricator_create_build_plan.png", alt: "Creating a new Build Plan in Phabricator" %>
+
+Provide a name then press "Create Build Plan".
+
+Now, add a step to the Build Plan. A "Build Step" calls one BuildKite
+"pipeline". If you need to call multiple pipelines then create multiple Build
+Steps.
+
+<%= image "phabricator_run_in_buildkite.png", alt: "Selecting Builkite for this Build Step" %>
+
+Press "Add Build Step" on the next screen. Then select "Build with Buildkite".
+
+<%= image "phabricator_blank_build_step.png", alt: "Incomplete Build Step" %>
+
+Keep this screen open while you configure Buildkite.
+
+## Step 2: Configure Buildkite Notification Webhook
+
+Phabricator and Buildkite integrate via webhooks. Phabricator triggers
+Buildkite via webhook, then Buildkite reports the status via webhook. You need
+to create an API Access Token in Buildkite to configure Phabricator.
+
+Open the [New API Access
+Token](https://buildkite.com/user/api-access-tokens/new) page in Buildkite.
+Provide a description, then select the relevant organization. Check
+`read_builds` and `write_builds`. Create the token then save a copy on the next
+screen.
+
+<%= image "buildkite_new_api_access_token.png", alt: "Creating a new Buildkite API Access Token" %>
+
+Next, configure the "Webhook Notification" for your Buildkite organization.
+
+Use the "Webhook URL" (it should end in `/harbormaster/hook/buildkite/`) under
+"Webhook Configuration" in Phabricator for "Webhook URL" in Buildkite.  Make a
+note of the autogenerated value in the "Token" field in Buildkite.  Check
+`build.finished` under Events, then save the notification settings.
+
+<%= image "buildkite_notification_webhook_settings.png", alt: "Buildkite Notification Webhook settings" %>
+
+Switch back to Phabricator.
+
+## Step 3: Complete Phabricator Build Step
+
+Press "Add New Credential" next to "API Token". Provide a name and use the
+Buildkite API Access Token from earlier for the "Token" field.
+
+<%= image "phabricator_new_credential.png", alt: "Creating a new Credential in Phabricator" %>
+
+Fill in the Buildkite "Organization Name" and "Pipeline Name". Use the Token
+from Builkite's Notfication Webhook for "Webhook Token" in Phabricator. Both
+should have the same value.
+
+<%= image "phabricator_build_step.png", alt: "Completing the Phabricator Build Step" %>
+
+Finally press "Create Build Step".
+
+## Step 4: Test with Manual Build
+
+Click "Run Plan Manually" on the next screen. Provide a revision ID from the repository inside Phabricator.
+
+<%= image "phabricator_manual_build_trigger.png", alt: "Run a manual build in Phabricator" %>
+
+Your Buildkite Pipeline should run then report back to Phabricator.
+
+<%= image "phabricator_build_status.png", alt: "Build status in Phabricator" %>
+<%= image "buildkite_pipeline.png", alt: "Build status in Phabricator" %>
+
+[phabricator]: https://phacility.com/phabricator/
```

Comment:
Could we add a bit to the end that says something about how it will now automatically run when they commit things?

Reply:
Done.